monkeys/helm/infinitemonkeys/values.yaml

```diff
@@ -0,0 +1,22 @@
+service:
+  appName: infinitemonkeys
+  controllerName: monkey-controller
+  serverName: monkey-server
+
+monkeyController:
+  name: Diya
+  teamName: Diya
+  instanceURL: diya.wire.link
+  userCount: 3
+  authUser: basic-auth-user
+  authPassword: "..."
+
+# replicaCount needs to be one more than userCount above
+# as monkeyController expects an additional owner
+monkeyServer:
```

Comment:
maybe this can be calculated with userCount +1 ?
so we don't need an additional parameter